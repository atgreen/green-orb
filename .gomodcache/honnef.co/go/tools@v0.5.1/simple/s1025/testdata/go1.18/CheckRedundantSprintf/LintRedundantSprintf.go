package pkg

import "fmt"

type MyByte byte
type T1 []MyByte

func fn() {
	var t1 T1
	_ = fmt.Sprintf("%s", t1) //@ diag(`underlying type is a slice of bytes`)
}
