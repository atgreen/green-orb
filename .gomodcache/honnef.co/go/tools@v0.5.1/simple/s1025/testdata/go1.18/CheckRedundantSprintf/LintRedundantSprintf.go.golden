package pkg

import "fmt"

type MyByte byte
type T1 []MyByte

func fn() {
	var t1 T1
	_ = string(t1) //@ diag(`underlying type is a slice of bytes`)
}
